flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceITCase.java

```diff
@@ -447,6 +447,51 @@ public void testProjectionPushDown() throws Exception {
         }
     }
 
+    @Test
+    public void testReadParquetNestedPushdown() throws Exception {
```

Comment:
Your changes also cover the lookup table source, so I think we should also add a lookup projection pushdown test?

Reply:
lookup projection pushdown is not supported when using Hive table as lookup table, but it is supported when using Hive table as scan table.